src/lib/types/model.ts

```diff
@@ -365,3 +365,66 @@ export interface IFeatureStrategySegment {
     featureStrategyId: string;
     segmentId: number;
 }
+
+export interface ISuggestChangeset {
+    id: number;
+    state: string;
+    project: string;
+    environment: string;
+    createdBy?: Pick<User, 'username' | 'imageUrl'>;
+    createdAt?: Date;
```

Comment:
Why are these two properties optional? Are there any situations where we'll have a change set without knowing who made it or when?

Reply:
see below